src/Bicep.MSBuild.E2eTests/src/csharp.test.ts

```diff
@@ -37,11 +37,11 @@ describe("msbuild", () => {
     expect(buildResult.stderr).toBe("");
 
     // both build and publish outputs should be present
-    example.expectTemplate(templateRelativePath);
-    example.expectTemplate(templateRelativePath2);
-    example.expectTemplate("bin/Debug/net7.0/publish/empty.json");
-    example.expectTemplate("bin/Debug/net7.0/publish/empty2.json");
-    example.expectNoFile("bin/Debug/net7.0/publish/skip.json");
-    example.expectNoFile("bin/Debug/net7.0/publish/skip2.json");
+    example.expectTemplate("bin/Release/net8.0/empty.json");
```

Comment:
>Release [](http://example.com/codeflow?start=32&length=7)

Interesting... So they changed the default project config to Release?

Reply:
Yup - see https://learn.microsoft.com/en-us/dotnet/core/compatibility/sdk/8.0/dotnet-publish-config